formulas/wick-base/functions/wick-make-group

```diff
@@ -28,10 +28,7 @@ wickMakeGroup() {
 
     if [[ -z "$group" ]]; then
         wickInfo "Creating group: $groupname" || return $?
-        args=(groupadd)
-
-        args[${#args[@]}]=$groupname
-        wickDebug "Command:" "${args[@]}" || return $?
+	args=(groupadd "$groupname")
```

Comment:
The new version of this line has the indentation incorrect. Are you using the EditorConfig plugin for your editor?